code/game/objects/random/medicine.dm

```diff
@@ -3,20 +3,55 @@
 	icon_state = "meds-green"
 
 /obj/random/medical/item_to_spawn()
-	return pickweight(list(/obj/item/stack/medical/bruise_pack = 4,\
-				/obj/item/stack/medical/ointment = 4,\
-				/obj/item/stack/medical/advanced/bruise_pack = 2,\
-				/obj/item/stack/medical/advanced/ointment = 2,\
-				/obj/item/stack/medical/splint = 1,\
-				/obj/item/bodybag = 2,\
-				/obj/item/bodybag/cryobag = 1,\
-				/obj/item/weapon/storage/pill_bottle/kelotane = 2,\
-				/obj/item/weapon/storage/pill_bottle/antitox = 2,\
-				/obj/item/weapon/storage/pill_bottle/tramadol = 2,\
+	return pickweight(list(/obj/item/stack/medical/bruise_pack = 5,\
+				/obj/item/stack/medical/ointment = 5,\
+				/obj/item/stack/medical/advanced/bruise_pack = 3,\
+				/obj/item/stack/medical/advanced/ointment = 3,\
+				/obj/item/stack/medical/splint = 2,\
+				/obj/item/bodybag = 3,\
+				/obj/item/bodybag/cryobag = 2,\
+				/obj/item/weapon/reagent_containers/glass/bottle/stoxin = 2,\
+				/obj/item/weapon/reagent_containers/glass/bottle/toxin = 2,\
+				/obj/item/weapon/reagent_containers/glass/bottle/cyanide = 0.5,\
+				/obj/item/weapon/reagent_containers/glass/bottle/mutagen = 0.5,\
+				/obj/item/weapon/reagent_containers/glass/bottle/ammonia = 2,\
+				/obj/item/weapon/reagent_containers/glass/bottle/pacid = 0.5,\
+				/obj/item/weapon/reagent_containers/glass/bottle/inaprovaline = 3,\
+				/obj/item/weapon/reagent_containers/syringe = 5,\
 				/obj/item/weapon/reagent_containers/syringe/antitoxin = 2,\
 				/obj/item/weapon/reagent_containers/syringe/antiviral = 1,\
 				/obj/item/weapon/reagent_containers/syringe/inaprovaline = 2,\
 				/obj/item/stack/nanopaste = 1))
+				
+/obj/random/medical/pill_bottle/random/maint
```

Comment:
This shouldn't be a subtype of medical, have two types:

/obj/random/pills 
/obj/random/pills/maint
put the postspawn only on the second one